adadamp/_dist.py

```diff
@@ -1,29 +1,53 @@
-from copy import copy, deepcopy
-from functools import partial
-from typing import Any, Callable, Dict, List, Optional, Union, Tuple, NewType
-
 import dask
 import numpy as np
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 import torch.optim as optim
+import random
 from torch.optim import Optimizer
-from time import time
+from time import sleep, time
 from distributed import get_client
 from sklearn.base import BaseEstimator
-from sklearn.exceptions import NotFittedError
 from sklearn.utils import check_random_state
-from skorch import NeuralNet
+from copy import copy, deepcopy
+from typing import Any, Callable, Dict, List, Optional, Union, Tuple, NewType
+from warnings import warn
 from torch.autograd import Variable
-from torch.utils.data import Dataset, IterableDataset, TensorDataset
-from torch.nn.modules.loss import _Loss as Loss
+from torch.utils.data import Dataset, IterableDataset, TensorDataset, DataLoader
+from adadamp.adadamp.dampers import SimpleBaseDamper, SimpleGeoDamp
```

Comment:
```suggestion
from .dampers import SimpleBaseDamper, SimpleGeoDamp
```

Reply:
It helps to run `pip install -e .` in the root of this repo.

Reply:
What does this do?

Reply:
`pip install -e /path/to/adadamp` installs the AdaDamp package. It looks like `from adadamp.adadamp.dampers import ...` is not using the adadamp package; instead, it looks like it's using a relative path.

It might cleaner to do `from adadamp.dampers import SimpleBaseDamper, SimpleGeoDamp`. That's what `from .dampers import` is doing (but is relative, not absolute). Here's more detail: https://realpython.com/absolute-vs-relative-python-imports/